test/unit/constructor.js

```diff
@@ -2,16 +2,16 @@ import { define } from '../../src/index';
 import helperElement from '../lib/element';
 import resolved from '../lib/resolved';
 
-describe('constructor', function () {
-  var id;
+describe('constructor', () => {
```

Comment:
we're using arrow-functions everywhere (including AtlasKit), since normal functions are being yelled at by the airbnb linter. I disabled lines where we're actually using `this`, not many of them.